source/server-discovery-and-monitoring/tests/rs/discover_arbiters.json

```diff
@@ -1,6 +1,6 @@
 {
-  "description": "Discover arbiters",
-  "uri": "mongodb://a/?replicaSet=rs",
```

Comment:
What's the reasoning for changing this from `replicaSet=rs` to `directConnection=false`? They test two different things (replica set name -> start in RSNP topology type, `directConnection=false` -> start in Unknown topology type) so they should either be two separate tests or we can just reset this to the original test because the expectations are still correct.

Reply:
I agree. The same question applies to a number of similar tests in the suite here.